lib/write.js

```diff
@@ -48,8 +48,10 @@ var Write = Juttle.proc.sink.extend({
     formatPoints: function(points) {
         _.each(points, function(pt) {
             _.each(pt, function(value, key) {
-                if (value instanceof JuttleMoment) {
+                if (values.isDate(value) || values.isDuration(value)) {
                     pt[key] = new Date(value.valueOf());
+                } else if (values.isObject(value) || values.isArray(value)) {
+                    pt[key] = JSON.stringify(value);
```

Comment:
And a slightly different answer :) Unlike Influx, I think it is a common scenario to store JSON into string columns of an SQL database (we actually did this in the original Jut frontend). I'd keep it here, for Influx, I'm still not sure :)